user_guide_src/source/concepts/index.rst

```diff
@@ -9,6 +9,7 @@ The following pages describe the architectural concepts behind CodeIgniter4:
 
     structure
     mvc
+    preloading
```

Comment:
Adding this in `concepts` seems strange.
Please move it to [installation](https://github.com/codeigniter4/CodeIgniter4/tree/develop/user_guide_src/source/installation).

In `4.5`, we have "Deployment", but it is not in `develop`.
https://github.com/codeigniter4/CodeIgniter4/blob/4.5/user_guide_src/source/installation/deployment.rst#php-preloading

Reply:
@kenjis Done move to Deployment and send to 4.5

Reply:
Thanks!